src/fixtures/vislib/mock_data/date_histogram/_rows.js

```diff
@@ -1,7 +1,8 @@
 import moment from 'moment';
 
 export default {
```

Comment:
As long as you are updating all of the fixtures, how about exporting a named object instead of default?